episodes/2-keras.Rmd

```diff
@@ -735,17 +740,15 @@ visualize a confusion matrix.
 
 :::: solution
 ## Solution
-The confusion matrix shows that the predictions for Adelie and Gentoo
-are decent, but could be improved. However, Chinstrap is not predicted
-ever.
-The training loss was very low, so from that perspective this may be
-surprising.
-But this illustrates very well why a test set is important when training
-neural networks.
-We can try many things to improve the performance from here.
-One of the first things we can try is to balance the dataset better.
-Other options include: changing the network architecture or changing the
-training parameters
+The confusion matrix shows that the predictions for Adelie and Gentoo are decent, but could be improved. However, Chinstrap is not predicted ever. Why are the Chinstraps not present in the confusion matrix from prediction?
+If we go back to the [**Pair Plot**](https://carpentries-incubator.github.io/deep-learning-intro/2-keras.html) in the Visualization section, we can figure it out that the biggest challenge is masking the Chinstrap penguins from the marginal distributions of the four features (bill length, bill depth, flipper length, and body mass). That is, there is no single variable that separates Chinstrap penguins from all other species. The combination of bill length and bill depth (the 2nd subfigure in the first row in the pairplot) gives a good separation of Chinstrap from Adelie and Gentoo penguins.
```

Comment:
My previous commit seems to have gotten lost among the other fixes, so just to keep this from getting lost: the link should be relative to make it independent of the specific server and path.